Readme.md

```diff
@@ -7,4 +9,22 @@ a low cost, open source Software Defined Radio platform.
 
 principal author: Michael Ossmann <mike@ossmann.com>
 
-http://greatscottgadgets.com/hackrf/
+Information on HackRF and purchasing HackRF: http://greatscottgadgets.com/hackrf/
+
+--------------------
+
+# Documentation
+
+Documentation for HackRF can be viewed on [Read the Docs](https://hackrf.readthedocs.io/en/latest/). 
+The raw documenation files for HackRF are in the [docs folder](https://github.com/mossmann/hackrf/tree/master/docs) 
+in this repository. Documentation changes can be submitted through pull request and suggestions can be made as 
+GitHub issues. 
+
+--------------------
+
+# Response Times
```

Comment:
Please change this header to "Getting Help" and add information from the GSG web site before the paragraph on response times.